nncf/experimental/onnx/graph/pattern_operations.py

```diff
@@ -0,0 +1,56 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from nncf.common.graph.patterns import merge_two_types_of_operations
+
+LINEAR_OPERATIONS = {'type': ['Conv',
```

Comment:
Could you use meta types here?

Reply:
We have a special ticket on implementing pattern matching algorithm based on metatypes- 58797.